HelpSource/scdoc.js

```diff
@@ -193,3 +193,35 @@ function fixTOC() {
         set_up_toc();
     }
 }
+
+// Set up a QWebChannel for communicating with C++ IDE objects. The main app publishes a handle
+// to IDE functionality at "IDE" which is made globally available here after the page and
+// WebSocket have loaded.
+function setUpWebChannel() {
+    if (location.search != "")
+        var baseUrl = (/[?&]webChannelBaseUrl=([A-Za-z0-9\-:/\.]+)/.exec(location.search)[1]);
```

Comment:
just drop the if-part all together! it's never going to be used since we hardcoded `ws://localhost:12345` in the server anyway. (it allows overriding the websocket url by setting a query string `?webChannelBaseUrl=...`.

Reply:
done